lib/blueprint_agreement/rspec/matcher_shall_agree_upon.rb

```diff
@@ -0,0 +1,7 @@
+require 'rspec/expectations'
+
+RSpec::Matchers.define :shall_agree_upon do |contract_name|
```

Comment:
Add the Rspec as a module block to prevent error when use is not using the Rspec gem

```
module Rspec do 
 ...
end 
```